pkg/resource/events-logger-config/grafana-agent/events-logger-config.grafanaagent.yaml.template

```diff
@@ -1,35 +1,17 @@
-logging {
-	level  = "info"
-	format = "logfmt"
-}
-
-loki.source.kubernetes_events "local" {
-	namespaces = {{ .ScrapedNamespaces }}
-	forward_to = [loki.write.default.receiver]
-}
-
-remote.kubernetes.secret "credentials" {
-	namespace = "kube-system"
-	name = "{{ .SecretName }}"
-}
-
-loki.write "default" {
-	endpoint {
-		url       = nonsensitive(remote.kubernetes.secret.credentials.data["logging-url"])
-		tenant_id = nonsensitive(remote.kubernetes.secret.credentials.data["logging-tenant-id"])
-
-		basic_auth {
-			username = nonsensitive(remote.kubernetes.secret.credentials.data["logging-username"])
-			password = remote.kubernetes.secret.credentials.data["logging-password"]
-		}
-
-		tls_config {
-			insecure_skip_verify = {{ .InsecureSkipVerify }}
-		}
-	}
-	external_labels = {
-		installation = "{{ .Installation }}",
-		cluster_id = "{{ .ClusterID }}",
-		scrape_job = "kubernetes-events",
-	}
-}
+# This file was generated by logging-operator.
+# It configures the Grafana-agent to be used as events logger.
+# - configMap is generated from events-logger.grafanaagent.template and passed as a string
+#   here and will be created by Grafana-agent's chart.
+# - Grafana-agent runs as a deployment, with only 1 replica.
+grafana-agent:
+  agent:
+    configMap:
+      content: |-
+        {{ .GrafanaAgentInnerConfig | nindent 8 }}
```

Comment:
```suggestion
        {{- .GrafanaAgentInnerConfig | nindent 8 }}
```

Reply:
You're right this just remove an extra Line in thé generated template

Reply:
oh ok, then why not :shrug: